src/libs/ajax.js

```diff
@@ -120,7 +120,7 @@ export const DAC = {
   delete: async (dacId) => {
     const url = `${await Config.getApiUrl()}/dac/${dacId}`;
     const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), { method: 'DELETE' }]));
-    return res.json();
+    return res;
```

Comment:
by not parsing the response to a json, we can just check the response status in handleDeleteDAC. the reason the handleDeleteDAC method wasn't working was because the response was returning successfully but then the res.json() line was throwing an unexpected end of input error (I think because there is nothing in the body) and that was triggering the catch phrase in handleDeleteDAC causing it to display an error even though the delete was successful

Reply:
also, the only place this is used is in handleDeleteDac on the ManageDac page

Reply:
We should add a comment here about why this is needed since it deviates from our standard pattern. Very much makes sense for API calls that don't have any response content.